api/v1alpha1/openapi.yaml

```diff
@@ -3756,8 +3756,36 @@ components:
           description: A list of application content.
           items:
             $ref: "#/components/schemas/ApplicationContent"
+        volumes:
```

Comment:
Can you add this for the ` ImageApplicationProviderSpec` - which is actually more useful in context of catalog right now.

Reply:
pushed